marine/marine.py

```diff
@@ -15,6 +15,13 @@ class MarineResult(Structure):
 
 
 class Marine:
+    SUGGESTED_MACROS = {
+        "macro.ip.src": ["ip.src", "arp.src.proto_ipv4"],
```

Comment:
ipv6?

Reply:
It's not a part of the macro because it isn't in the current version of marine.